data/hai/unfettered 0 prologue.txt

```diff
@@ -297,6 +297,106 @@ mission "Unfettered Jump Drive 3"
+				goto meet
+			label grab
+			`	You manage to grab the weapon, just in time to deflect an incoming attack from the same Unfettered. You duel with them for what feels like an eternity, and you are astonished by the fact you managed to survive that long. Then, suddenly, they smile, and lower their weapon. "You sure know how to fight! I am Desneeparu, and it is pleasure to meet you, captain <last>!" She says, before laughing at your surprised face.`
+			label meet
+			action
+				log "minor people" "Desneeparu" `An Unfettered warrior that knows how to handle a blade. She fought you to test your resolve before you were given access to their weapons. You suspect she was holding back or you would never have stood a chance.`
+			`	Now that you have successfully passed the test, you are then taken to an underground zone, passing trough what looks like very basic weapon testing. Many Hai are there, doing small modifications and testing them, but never going really far in the tests. Seeing your interest, the engineer explains. "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually shoots before Unfettered captain can get their hands on it." Now that they mention it you notice a few captains looking at the tests, and one even requests a weapon to be taken to his ship, apparently satisfied with its performances. You keep on walking until you enter a proper outfitter section of the facility.`
+			`	"Welcome to the special section of the outfitter, captain! It is only for our most distinguished guests. Now that you have sold us multiple jump drives, I can tell you that we can offer something way more valuable than money. For now, I can only offer you the mighty tripulse shredder, but eventually we will give you access to all kinds of weapons. 2 tripulse cannons for one jump drive and our sympathy is the deal I propose." They say, presenting the weapons to you. Now that you can take a closer look, you notice these weapons seem fairly similar to the rest of Hai weaponry, but enhanced someway or another. There even are some ion looking weapons, but one is smaller than an ion cannon whilst another one looks like a turreted version of it.`
+			label weapons
+			choice
+				`	"Tell me more about that tripulse cannon I can buy."`
+					goto tripulse
+				`	"Can you tell me more about that ion looking small weapon?"`
+				`	"I'd like to learn about the ion turret."`
+					goto turret
+				`	"I would just prefer the money, actually."`
+					goto money
+			label blaster
+			`	"Ah, the blaster. It is a favored by many Shield Beetle captains, because it can easily help them turn the tide of battle, without needing to dedicate too much weapon space for it. It is true that it does not do enough ion damage to properly disable the enemy ship, but it is able to jam their weapons, thus giving you the upper hand in a long fight, especially since its damage potential is acceptable."`
```

Comment:
```suggestion
			`	"Ah, the Blaster. It is a favored by many Shield Beetle captains, because it can easily help them turn the tide of battle, without needing to dedicate too much weapon space for it. It is true that it does not do enough ion damage to properly disable the enemy ship, but it is able to jam their weapons and give you the upper hand in a long fight, especially since its damage potential is acceptable."`
```